rfc/0075-faster-failover-and-configuration-push.md

```diff
@@ -0,0 +1,337 @@
+    Opcode       (1)    : 0xb5
+    Key length   (2,3)  : 0x0000
+    Extra length (4)    : 0x00
+    Data type    (5)    : 0x00 (RAW)
+    Vbucket      (6,7)  : 0x0000
+    Total body   (8-11) : 0x00000010 (16 bytes)
+    Opaque       (12-15): 0xdeadbeef
+    CAS          (16-23): 0x0000000000000000
+    Epoch        (24-31): 0x0000000000000042 (66 in base-10)
+    Revision     (32-39): 0x0102030405060708 (72623859790382856 in base-10)
+
+If the node has a cluster configuration newer than what is specified in the example, the response will include the new
+configuration in the body with the data type set to `JSON` (`0x01`). Otherwise, the response will have an empty body
+with the data type `RAW` (`0x00`).
+
+### Deduplicate Cluster Configuration for `NotMyVbucket` Responses
+
+[https://review.couchbase.org/c/kv_engine/+/190899]: #
+
+The KV engine introduces a new HELLO flag called `DedupeNotMyVbucketClustermap` with a value of `0x1e`. Once this flag
+is negotiated, the node might send an empty body with `NotMyVbucket` (`0x07`) status codes. The KV engine tracks the
+revision that has been sent to the SDK over the socket connection, so a response with a `NotMyVbucket` status will only
+have a body if the pushed version is older than the active configuration.
+
+The KV engine updates the pushed configuration version in the following cases:
+* Configuration sent to the SDK in response to a `GetClusterConfig` (`0xb5`) request.
+* Configuration pushed to the SDK that enabled the HELLO flag `ClustermapChangeNotification` (`0x0d`).
+
+NOTE: `DedupeNotMyVbucketClustermap` affects `ClustermapChangeNotification` and `ClustermapChangeNotificationBrief`
+features, that described below. In other words, if deduplication enabled, the cluster configuration will be announce for
+the socket connection only once.
+
+NOTE: KV engine does not inspect the body of the configuration, it only looks at `epoch`/`revision` pair to detect
+duplicates (decide if the SDK seen the update). It means, that SDK still might receive updates that do not change
+VBucket map or topology (number, order or names of the nodes). It is up to the SDK to decide if the configuration
+affects the sockets or pipeline configuration, which must be already implemented for older generation of the servers.
+
+### Enforcing Snappy Compression for Cluster Configuration Payloads
+
+[https://review.couchbase.org/c/kv_engine/+/192152]: #
+[https://review.couchbase.org/c/kv_engine/+/192316]: #
+
+The KV engine introduces a new HELLO flag called `SnappyEverywhere` with a value of `0x13`. Once this flag is
+negotiated, the node will always use the compressed version of the cluster configuration and data type flags will be set
+to `JSON | SNAPPY` (`0x03`).
+
+Note, that the meaning of the flag `SnappyEverywhere` is that SDK expects and properly handles compression for **ANY**
+operation during communication with the KV engine, this is why the flag called "SnappyEverywhere", and "SnappyConfig".
```

Comment:
typo: and "SnappyConfig" -> and **not** "SnappyConfig"

Reply:
Fixed